src/shared/view/nunjucks/macros/contact-details.njk

```diff
@@ -1,13 +1,27 @@
-{% macro contactDetails() %}
+{% macro contactDetails(emailAddress, isSingleSpaced) %}
```

Comment:
I think it might be nicer if the signature for this macro is something like

```JavaScript
{% macro contactDetails(options = {
  email: 'enquiries@environment-agency.gov.uk',
  isSingleSpaced: false 
}) %}
```

This means that in almost all of the macro's use it will be a simple bit of markup like

```
{{ contactDetails() }}
```
or
```
{{ contactDetails({ isSingleSpaced: true ) }}
```

with one instance of:
```
{{
  contactDetails({
    email: 'national-operator-returns@environment-agency.gov.uk',
    isSingleSpaced: true 
  })
}}
```